src/components/organisms/Sider/Sider.styled.tsx

```diff
@@ -114,3 +116,33 @@ export const StyledOtherItem = styled.div`
     }
   }
 `;
+
+export const DropdownListItem = styled.li`
+  display: flex;
+  align-items: center;
+
+  height: 50px;
+
+  cursor: pointer;
+  transition: 0.3s;
+
+  a {
+    flex: 1;
+
+    padding: 14px;
+  }
+
+  &:hover {
+    .testkube-text {
+      color: ${Colors.whitePure};
+    }
+
+    svg {
+      path {
+        fill: ${Colors.whitePure};
+
+        transition: 0.3s;
+      }
+    }
+  }
```

Comment:
I'm not sure if that's the case here, but if everything in `a` has to be same color:
```suggestion
  &:hover {
    color: ${Colors.whitePure};

    .testkube-text {
      color: inherit;
    }

    svg {
      path {
        fill: currentColor;
        transition: 0.3s;
      }
    }
  }
```

Reply:
Checked, does not work as expected

https://github.com/kubeshop/testkube-dashboard/assets/25991946/5299b8aa-4238-4e8f-96a6-779133c8b80b